int8 whether_start
---
string result
string fail_reason 
